DOCS.md

```diff
@@ -130,3 +130,38 @@ Cannot connect to the Docker daemon. Is 'docker -d' running on this host?
 ```
 
 The above issue can be resolved by setting `storage_driver: vfs` in the `.drone.yml` file. This may work, but will have very poor performance as discussed [here](https://github.com/rancher/docker-from-scratch/issues/20).
+
+
+This error occurs when using Debian wheezy or jessie which kernel misses some features:
+
+```
+time="2015-12-17T08:06:57Z" level=debug msg="Mounting none /sys/fs/cgroup/blkio cgroup blkio" 
+time="2015-12-17T08:06:57Z" level=debug msg="Mounting none /sys/fs/cgroup/perf_event cgroup perf_event" 
+time="2015-12-17T08:06:57Z" level=debug msg="Mounting none /sys/fs/cgroup/cpuset cgroup cpuset" 
+time="2015-12-17T08:06:57Z" level=debug msg="Mounting none /sys/fs/cgroup/cpu,cpuacct cgroup cpu,cpuacct" 
+time="2015-12-17T08:06:57Z" level=debug msg="Creating /sys/fs/cgroup/memory" 
+time="2015-12-17T08:06:57Z" level=debug msg="Mounting none /sys/fs/cgroup/memory cgroup memory" 
+time="2015-12-17T08:06:57Z" level=fatal msg="no such file or directory" 
+```
+
+The above issue can be resolved by editing your grub.cfg and adding these options to you kernel image:
```

Comment:
The above issue can be resolved by editing your `grub.cfg` and adding these options to your kernel image: